packages/infra/lib/api-stack/api-service.ts

```diff
@@ -164,6 +164,7 @@ export function createAPIService(
           ...(cookieStore && {
             CW_MANAGEMENT_COOKIE_SECRET_ARN: cookieStore.secretArn,
           }),
+          CQ_API_KEY: props.config.carequality.cqApiKey,
```

Comment:
this won't work - this should be in `secrets`, right?

we need to update the secrets stack as such

this isn't just an env var, it's a secret